test/V1/DiamondProxy/DisputeManager/Timeout.test.ts

```diff
@@ -63,213 +63,8 @@ describe("DisputeManagerProxy", function () {
     });
 
     describe("Timeout", function () {
-        it("Timeout - transaction 0", async function () {
```

Comment:
why?
- restore the tests
- use `it.skip` if they are failing, write note about it
- adept the tests to the new impl and make them pass